frappe/__init__.py

```diff
@@ -52,8 +52,30 @@ def _(msg, lang=None):
 	if lang == "en":
 		return msg
 
-	return get_full_dict(local.lang).get(msg) or msg
+	# To parse html data
+	msg = parse_msg(msg)
```

Comment:
what is the use-case for this?

Also only call this if `<` is present in the string. otherwise parsing will be expensive